clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/gvcf/TransferGvcfV1Metadata.scala

```diff
@@ -0,0 +1,44 @@
+package org.broadinstitute.clio.transfer.model.gvcf
+
+import java.time.OffsetDateTime
+
+import org.broadinstitute.clio.transfer.model.TransferMetadata
+import org.broadinstitute.clio.util.model.DocumentStatus
+
+case class TransferGvcfV1Metadata(analysisDate: Option[OffsetDateTime] = None,
+                                  contamination: Option[Float] = None,
+                                  documentStatus: Option[DocumentStatus] = None,
+                                  gvcfMd5: Option[String] = None,
+                                  gvcfPath: Option[String] = None,
+                                  gvcfSize: Option[Long] = None,
+                                  gvcfIndexPath: Option[String] = None,
+                                  gvcfMetricsPath: Option[String] = None,
+                                  pipelineVersion: Option[String] = None,
+                                  notes: Option[String] = None)
+    extends TransferMetadata[TransferGvcfV1Metadata] {
+
+  override def pathsToMove: Seq[String] =
+    Seq.concat(gvcfPath, gvcfIndexPath, gvcfMetricsPath)
+
+  override def rebaseOnto(destination: String): TransferGvcfV1Metadata = {
+    assert(destination.endsWith("/"))
```

Comment:
IMO I think we should only use `assert` in non-test code. If we want to validate input, it should explicitly throw an exception (or return `Option[]`)